core/src/main/clojure/xtdb/pgwire.clj

```diff
@@ -1554,6 +1556,7 @@
       :canned-response (cmd-write-canned-response conn canned-response)
       :set-session-parameter (cmd-set-session-parameter conn parameter value)
       :set-session-characteristics (cmd-set-session-characteristics conn session-characteristics)
+      :set-role nil
```

Comment:
@mbutlerw Is this the right way to add a no-op statement, for which we don't have an implementation yet?

Reply:
Not sure there's an established pattern yet, this seems like a reasonable approach for now.